<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPlot</class>
 <widget class="QWidget" name="FormPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,9">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>209</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="hLay14bit">
       <item>
        <widget class="QLabel" name="label14">
         <property name="text">
          <string>offset_14bit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox14Offset">
         <property name="minimum">
          <number>-10</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLay24bit">
       <item>
        <widget class="QLabel" name="label24">
         <property name="text">
          <string>offset_24bit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox24Offset">
         <property name="minimum">
          <number>-10</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="tBtnSimulate">
     <property name="text">
      <string>simulate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="tBtn3D">
     <property name="text">
      <string>3D</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="tBtnZoom">
     <property name="text">
      <string>zoom</string>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::ZoomFitBest"/>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="tBtnData">
     <property name="text">
      <string>data</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QToolButton" name="tBtnHistory">
     <property name="text">
      <string>history</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
